src/Components/Details/Details.tsx

```diff
@@ -87,9 +102,35 @@ const Details = ({
       .catch((error) => {
         console.log(`Error loading campground details ${error}`);
       });
+
+    getCampgroundReviews(selectedCampground)
+      .then((response) => {
+        if (response) {
+          setCampgroundReviews(formatReviews(response.data));
+        }
+      })
+      .catch((error) => {
+        console.log(`Error loading campground reviews ${error}`);
```

Comment:
In an ideal world, the user can see something on the page when an error is caught.